quickstart/nodejs/src/index.ts

```diff
@@ -0,0 +1,59 @@
+
+import { DefaultAzureCredential } from '@azure/identity';
+import { EntraIdCredentialsProviderFactory, REDIS_SCOPE_DEFAULT } from '@redis/entraid';
+import { createClient } from '@redis/client';
+
+const resourceEndpoint = process.env.AZURE_MANAGED_REDIS_HOST_NAME!;
+if (!resourceEndpoint) {
+    console.error('AZURE_MANAGED_REDIS_HOST_NAME is not set. It should look like: rediss://YOUR-RESOURCE_NAME.redis.cache.windows.net:<YOUR-RESOURCE-PORT>. Find the endpoint in the Azure portal.');
+}
+
+function getClient() {

```

Comment:
I'd recommend moving the contents of this function into the `try` block below so any errors are output to the console via the `catch`